source/content/guides/wordpress-configurations/05-wordpress-cache-plugin.md

```diff
@@ -39,12 +39,22 @@ You can increase the default time to live value to improve the chances that a vi
 
 1. Modify the **Default Cache Time**.
 
-    You should strike a balance between freshness of content and speed. We recommend a minimum of 600 seconds. If you can increase the setting to 30 minutes (1800 seconds) or 1 hour (3600 seconds), many more requests will hit the Edge Cache. Every page served from the Edge Cache won't hit your application container's PHP workers or MySQL database, which means faster page load times and a better user experience for site visitors.
+    You should strike a balance between freshness of content and speed. We recommend a minimum of 600 seconds. If you can increase the setting to 1 hour (3600 seconds) or even 1 week (604800 seconds), many more requests will hit the Edge Cache. Every page served from the Edge Cache won't hit your application container's PHP workers or MySQL database, which means faster page load times and a better user experience for site visitors.
```

Comment:
The premise of this first sentence is misguided. With PAPC, content should always be fresh since it's cleared. 

Let's explain that, and recommend 1 week. No need to mention 600 seconds or 1 hour.